jetpack-io/devbox
# [devbox global] Added init hook flag to global shellenv

## Summary
Added `--init-hook` flag to devbox global shellenv

## How was it tested?
- `./devbox global add hello`
- edit global devbox.json `code ~/.local/share/devbox/global/default/devbox.json` and add init_hook
- `./devbox global shellenv --init-hook`
- confirm the line `. ~/.local/share/devbox/global/default/.devbox/gen/scripts/.hooks.sh` is included in the output.
- `cat ~/.local/share/devbox/global/default/.devbox/gen/scripts/.hooks.sh` and confirm the init_hook is included in the file.
